lametro/wagtail_hooks.py

```diff
@@ -230,10 +231,50 @@ class FiscalYearCalendarViewSet(SnippetViewSet):
     )
 
 
+class TooltipFilterSet(django_filters.FilterSet):
```

Comment:
Following a pattern similar to what we did with alerts, the tooltips will default to showing only those that are enabled, and the filters can adjust from there.